src/weights.jl

```diff
@@ -313,6 +319,12 @@ length(wv::UnitWeights) = wv.len
 size(wv::UnitWeights) = tuple(length(wv))
 Base.axes(wv::UnitWeights) = tuple(Base.OneTo(length(wv)))
 
+# https://github.com/JuliaLang/julia/pull/43354
+if VERSION >= v"1.8.0-DEV.1494" # 98e60ffb11ee431e462b092b48a31a1204bd263d
```

Comment:
Not a big deal at all but it might be nice to combine the two conditionals so that there's only one place that needs editing once we require 1.8 or later.

Reply:
Fixed in https://github.com/JuliaStats/StatsBase.jl/pull/894/commits/f2421a46af446f90bdb9bad3173f6dfd249f3f45.